burn-train/src/metric/learning_rate.rs

```diff
@@ -0,0 +1,48 @@
+use super::{
+    state::{FormatOptions, NumericMetricState},
+    MetricMetadata, Numeric,
+};
+use crate::metric::{Metric, MetricEntry};
+
+/// Track the learning rate across iterations.
+pub struct LearningRateMetric {
+    state: NumericMetricState,
+}
+
+impl LearningRateMetric {
+    pub fn new() -> Self {
+        Self {
+            state: NumericMetricState::new(),
+        }
+    }
+}
+
+impl Default for LearningRateMetric {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl Metric for LearningRateMetric {
+    type Input = ();
+
+    fn update(&mut self, _item: &(), metadata: &MetricMetadata) -> MetricEntry {
+        let learning_rate = metadata.learning_rate.unwrap_or(0.0);
+
+        self.state.update(
+            learning_rate,
+            1,
+            FormatOptions::new("Learning Rate").precision(2),
+        )
+    }
+
+    fn clear(&mut self) {
+        self.state.reset()
+    }
+}
+
+impl Numeric for LearningRateMetric {
+    fn value(&self) -> f64 {
```

Comment:
for consistency, should use `LearningRate` instead of 64.

Reply:
Same here, it's just a numeric value